var/spack/repos/builtin/packages/sherpa/package.py

```diff
@@ -159,10 +159,12 @@ class Sherpa(AutotoolsPackage):
 
     # Note that the delphes integration seems utterly broken: https://sherpa.hepforge.org/trac/ticket/305
 
-    depends_on("autoconf", type="build")
-    depends_on("automake", type="build")
-    depends_on("libtool", type="build")
-    depends_on("m4", type="build")
+    # autotools dependencies are needed at runtime to compile processes
+    # at least as long as sherpa is an autotools package
+    depends_on("autoconf")
+    depends_on("automake")
+    depends_on("libtool")
+    depends_on("m4")
```

Comment:
From the comments here and on the key4hep issue, it should only be libtool that's needed at runtime, right?

Reply:
What is done at runtime is to do a full configure - compilation cycle over generated processes so I think all the tools are needed.